runner/execer/os/os_execer.go

```diff
@@ -15,13 +15,23 @@ func NewExecer() execer.Execer {
 
 type osExecer struct{}
 
+type WriterDelegater interface {
```

Comment:
0) copied the comment
1) no, I'm not sure it'll be useful then, but I suspect it will be, because...
2) we may still want to write to a file and then have out code in Go tail from the file, which would prevent our Go program from blocking the underlying process
4) it not necessarily being useful is why I didn't add it to Output itself, but just to one implementation